products/forms.py

```diff
@@ -1 +1,77 @@
 from django import forms
+from .models import Product, Category, Brand
+from .messages import BrandFormErrorMessages
+
+
+class AddProductForm(forms.ModelForm):
+    class Meta:
+        model = Product
+        fields = ['name', 'description', 'quantity', 'price', 'image', 'category', 'is_active']
+
+
+class AddCategoryForm(forms.ModelForm):
+    class Meta:
+        model = Category
```

Comment:
@Yshah112 add doc string for this form class

Reply:
done